scripts/ci/ci_mypy.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+set -euo pipefail
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=./_check_not_in_container.sh
+. "${MY_DIR}/_check_not_in_container.sh"
+
+pushd "${MY_DIR}/../../" || exit 1
+
+export AIRFLOW_CONTAINER_SKIP_SLIM_CI_IMAGE="false"
+export AIRFLOW_CONTAINER_SKIP_CI_IMAGE="true"
+export AIRFLOW_CONTAINER_PUSH_IMAGES="false"
+export AIRFLOW_CONTAINER_BUILD_NPM="false"
+export AIRFLOW_CONTAINER_CI_OPTIMISED_BUILD="true"
+
+# shellcheck source=../../hooks/build
+. ./hooks/build
+
+set -x
+docker run --entrypoint mypy "${AIRFLOW_SLIM_CI_IMAGE}" airflow tests
+set +x
+
+popd || exit 1
```

Comment:
Not that it matters, but unless this file is `source`d then it doesn't matter what CWD the script finishes up in.

Reply:
I know. But I prefer to always keep matching pairs of pushd/popd in my scripts - I like "bracketing". Just in case the script is copied to a bigger script.  I've been bitten by it several times in the past. It's a bit over the top but I think it's harmless :)

Reply:
I hope it's resolved :)